docs/proposals/network-policy.md

```diff
@@ -197,6 +254,35 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString `json:"port,omitempty"`
 }
 
+type NetworkCIDRRange struct {
+	// QUESTION: we could just do "Start" and "End" without a prefix to
+	// specify the range.  While this provides a simpler structure, it may
+	// be more error-prone since its less easily validated, and isn't very
+	// nice for IPv6 addresses.
```

Comment:
The "isn't very nice for IPv6 addresses" comment applied to any solution with only "allow" rules and no "deny" rules. It's not specific to End instead of Prefix.